lib/BloqadeLattices/src/bounded_lattice.jl

```diff
@@ -26,7 +26,7 @@ end
 
 
 
-function parallelepiped_region(lattice::AbstractLattice{D},M::Vararg{NTuple{D,Int},D};PBC::Bool=false) where D
+function parallelepiped_region(lattice::AbstractLattice{D},M::Vararg{NTuple{D,Real},D};PBC::Bool=false) where D
```

Comment:
The vectors defining the bounded region can be tuples of Integers OR Floats, so I felt it suitable to just define the type as `Real`